typings/index.d.ts

```diff
@@ -229,17 +228,17 @@ declare module 'discord.js' {
 
 	export class ClientApplication extends Base {
 		constructor(client: Client, data: object);
-		public botPublic?: boolean;
-		public botRequireCodeGrant?: boolean;
-		public cover?: string;
+		public botPublic: boolean | null;
+		public botRequireCodeGrant: boolean | null;
+		public cover: string | null;
 		public readonly createdAt: Date;
 		public readonly createdTimestamp: number;
 		public description: string;
 		public icon: string;
 		public id: Snowflake;
 		public name: string;
-		public owner?: User;
-		public rpcOrigins: string[];
+		public owner: User | null;
+		public rpcOrigins: string[] | [];
```

Comment:
An empty array can still be type of `string[]`, there's no need for `[]`.
```suggestion
		public rpcOrigins: string[];
```